io.ente.photos.yml

```diff
@@ -0,0 +1,44 @@
+id: io.ente.photos
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+rename-desktop-file: ente.desktop
+rename-icon: ente
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: ente
```

Comment:
is the icon on desktop entry working for you? Something seems to be wrong with the icon file itself because GNOME cannot display the icon.

Reply:
ok nevermind it worked on a reinstall.